anatomy

```diff
@@ -2334,14 +2334,14 @@ look under there, we'll see the following:
 $ ls -F usr/src/man
 Makefile        man3dat/        man3lgrp/       man3rpc/        man3xnet/
 Makefile.man    man3devid/      man3lib/        man3rsm/        man4/
-man1/           man3devinfo/    man3mail/       man3sasl/       man5/
-man1b/          man3dlpi/       man3malloc/     man3scf/        man7/
-man1c/          man3dns_sd/     man3mp/         man3sec/        man7d/
-man1has/        man3elf/        man3mpapi/      man3secdb/      man7fs/
-man1m/          man3exacct/     man3nsl/        man3sip/        man7i/
-man2/           man3ext/        man3nvpair/     man3slp/        man7ipp/
-man3/           man3fcoe/       man3pam/        man3socket/     man7m/
-man3bsm/        man3fstyp/      man3papi/       man3stmf/       man7p/
+man1/           man3devinfo/    man3mail/       man3sasl/       man4d/
+man1b/          man3dlpi/       man3malloc/     man3scf/        man4fs/
+man1c/          man3dns_sd/     man3mp/         man3sec/        man4i/
+man1has/        man3elf/        man3mpapi/      man3secdb/      man4ipp/
+man1m/          man3exacct/     man3nsl/        man3sip/        man4m/
```

Comment:
My understanding of IPD4 is that the entirety of gate's 1M is moving to section 8.

Reply:
Yes, I was trying to be clever and just manually redo it. I just went and did a new ls of everything. Thanks for catching that.